script/ghIssues.pl

```diff
@@ -74,8 +73,8 @@ sub getIssues{
 				repo => $repo || '',
 				issue_id => $issue->{'number'} || '',
                 author => $issue->{user}->{login} || '', 
-				title => decode_utf8($issue->{'title'}) || '',
-				body => decode_utf8($issue->{'body'}) || '',
+				title => $issue->{'title'} || '',
+				body => $issue->{'body'} || '',
```

Comment:
@kablamo So what this means is this new New::Github version automatically gets values decoded from utf8?